doc/CHEATSHEET.md

```diff
@@ -417,7 +410,7 @@ foo = do
     ...
     searchResults <-
       runUtxoSearch $
-        allUtxos `filterWithPred` ((== ada 10) . txSkelOutValue)
+        allUtxos `filterWithPred` ((== ada 10) . view txSkelOutValueL)
```

Comment:
Unrelated to this PR, just a passing remark:

Usually, `filter` functions have the function as first parameter (e.g. https://hackage.haskell.org/package/base-4.21.0.0/docs/Prelude.html#v:filter)

Shouldn't we stick to this convention too?

Reply:
That's a good point, I believe we had it this way for a while. 
I intend to dive into utxo searches later on, maybe even remove them altogether, we can change this at that time.